frame/dvm/ethereum/src/lib.rs

```diff
@@ -386,13 +375,13 @@ pub mod pallet {
 	#[pallet::storage]
 	#[pallet::getter(fn get_ring_remaining_balances)]
 	pub(super) type RemainingRingBalance<T: Config> =
-		StorageMap<_, Blake2_128Concat, T::AccountId, RingBalance<T>, ValueQuery>;
+		StorageMap<_, Blake2_128Concat, T::AccountId, u128, ValueQuery>;
```

Comment:
Please be careful about this when reviewing. I think no migration need here because the type is the same.

Reply:
Recommend using `Currency::Balance`

Reply:
How to do that? I have deleted the `RingCurrency` and `KtonCurrency` in the `Config`.

Reply:
Never mind, I think no any other projects will use this pallet.